src/compas/robots/model/joint.py

```diff
@@ -698,10 +695,8 @@ def transform(self, transformation):
         -------
         None
         """
-        if self.origin:
-            self.origin.transform(transformation)
-        if self.axis:
-            self.axis.transform(transformation)
+        self.current_origin.transform(transformation)
+        self.axis.transform(transformation)
```

Comment:
Shouldn't we do the same for `self.axis`?

Reply:
yes